### PR TITLE
Style Upgrades (Part 1)

### PR DESCRIPTION
**Pull request by SeanLeRoy**
**Description**
Style changes from Lyndsay :) shouldn't change any real business logic

**Testing**
Just been clicking around on web and on desktop on MacOS

### CODE REVIEW

File: packages/core/components/QuerySidebar/QuerySidebar.module.css [hunk 1]
```diff
@@ -81,30 +112,27 @@
...
     height: 50px;
     left: 0;
     max-width: var(--query-sidebar-max-width);
     position: absolute;
-    width: calc(0.3 * 80% - var(--margin));
+    width: 100%;
     z-index: 2;
 }
 
 .minimize-bar {
```
Comment (aswallace) [inline packages/core/components/QuerySidebar/QuerySidebar.module.css:133]: Can't tell if there's some padding missing on the minimize bar or if it's just the scrollbar being weird
image

File: packages/core/components/FileDetails/FileDetails.module.css [hunk 2]
```diff
@@ -1,62 +1,22 @@
...
-    height: 50px;
-    right: 0;
-    max-width: 100%;
-    position: absolute;
-    width: 100%;
-    z-index: 11
+    min-width: 175px;
+    max-width: 95%;
+    overflow: auto;
+    width: var(--file-details-width); /* defined in App.module.css under .root{} */
```
Comment (aswallace) [inline packages/core/components/FileDetails/FileDetails.module.css:13]: Getting some odd resizing behavior on this component where the contents resize but not the panel itself
image

File: packages/core/components/QueryPart/QueryPartRow.module.css [hunk 3]
```diff
@@ -32,19 +50,25 @@
 
 .row-title {
     display: flex;
-    width: calc(100% - 33px);
+    max-width: calc(100% - 33px);
+    width: 100%;
```
Comment (aswallace) [inline packages/core/components/QueryPart/QueryPartRow.module.css:54]: Some vertical misalignment for title vs icon. May be the same issue as the comment I put for filter
image

File: packages/core/components/FileDetails/FileDetails.module.css [hunk 4]
```diff
@@ -173,19 +168,9 @@
...
 .title {
-    font-size: large;
     margin: 0 0 0 var(--margin);
 }
 
 .icon-button span {
     font-weight: normal;
 }
-
-.icon-button:hover, .icon-button:active {
```
Comment (aswallace) [inline packages/core/components/FileDetails/FileDetails.module.css:184]: Since these styles are removed, the hover states are misbehaving
image
image

File: packages/core/components/FileDetails/index.tsx [hunk 5]
```diff
@@ -76,79 +67,34 @@ function resizeHandleDoubleClick() {
...
                                             className={styles.thumbnail}
-                                            width={thumbnailWidth}
-                                            height={thumbnailHeight}
-                                            uri={thumbnailPath}
+                                            width="100%"
+                                            // height={thumbnailHeight}
+                                            uri={fileDetails?.getPathToThumbnail()}
                                         />
                                     </div>
                                     <div className={styles.fileActions}>
```
Comment (aswallace) [inline packages/core/components/FileDetails/index.tsx:100]: Since we can't fully minimize this panel anymore, we might want to add some responsiveness to these action buttons? E.g., stacking
image

File: packages/core/components/TutorialTooltip/TutorialTooltip.module.css [hunk 6]
```diff
@@ -5,46 +5,65 @@
...
+    font-weight: 500;
+    margin: 0;
+    padding: 0
 }
 
-.tutorial-container {
+.tutorial-container > div {
     background-color: var(--primary-background-color);
+    border-radius: var(--small-border-radius);
+    color: var(--primary-text-color);
```
Comment (aswallace) [inline packages/core/components/TutorialTooltip/TutorialTooltip.module.css:38]: I think the tutorials are occasionally getting cut off a little on the far edges 
image

File: packages/web/src/components/Header/Header.module.css [hunk 7]
```diff
@@ -49,8 +47,6 @@
   color: var(--aqua);
   border: 1px solid var(--aqua);
   border-radius: 18px;
-  height: 30px;
-  width: 110px;
 }
```
Comment (aswallace) [inline packages/web/src/components/Header/Header.module.css:50]: May not be part of this PR, but missing some padding between header buttons 
image

File: packages/desktop/src/renderer/renderer.module.css
Comment (aswallace) [inline packages/desktop/src/renderer/renderer.module.css:1]: empty file?

**Review by aswallace — APPROVED** [review]
Left some nit comments but otherwise lgtm!

File: packages/core/styles/global.css [hunk 8]
```diff
@@ -21,59 +20,44 @@ main {
     padding: 0;
     border: 0;
     overflow: hidden;
-    scrollbar-color:  var(--borders-light-grey) var(--accent-dark)
+    scrollbar-color:  var(--medium-grey) var(--accent-dark)
```
Comment (aswallace) [inline packages/core/styles/global.css:23]: Just noticed this doesn't seem to get applied for me on desktop
image

File: packages/core/components/GlobalActionButtonRow/index.tsx [hunk 9]
```diff
@@ -66,43 +79,31 @@ export default function GlobalActionButtonRow(props: Props) {
...
-                    })}
-                    disabled={!shouldDisplayThumbnailView}
-                    onClick={() =>
-                        dispatch(
-                            selection.actions.setFileThumbnailView(!shouldDisplayThumbnailView)
-                        )
-                    }
-                    title="List view"
-                    iconProps={{ iconName: "BulletedList" }}
+                    iconProps={{ iconName: "Photo2", className: styles.smallFont }}
```
[the remark this answers is not in the document]
Comment (SeanLeRoy) [reply packages/core/components/GlobalActionButtonRow/index.tsx:82]: This specific one she recommended a "three line icon" 🤷  unsure why this one specifically i like either

File: packages/core/components/FileList/index.tsx [hunk 10]
```diff
@@ -37,8 +37,8 @@ const DEFAULTS = {
 };
 
 const MAX_NON_ROOT_HEIGHT = 300;
-const SMALL_ROW_HEIGHT = 18;
-const TALL_ROW_HEIGHT = 22;
+const SMALL_ROW_HEIGHT = 17;
+const TALL_ROW_HEIGHT = 19;
```
[the remark this answers is not in the document]
Comment (SeanLeRoy) [reply packages/core/components/FileList/index.tsx:41]: Yes! This is coming in part 2

File: packages/core/components/QuerySidebar/QuerySidebar.module.css (hunk 1 above)
Comment (SeanLeRoy) [reply packages/core/components/QuerySidebar/QuerySidebar.module.css:133]: This will be removed / altered in part 2 majorly

File: packages/core/components/FileDetails/FileDetails.module.css (hunk 2 above)
Comment (SeanLeRoy) [reply packages/core/components/FileDetails/FileDetails.module.css:13]: Are you shrinking the window or..?

File: packages/core/components/GlobalActionButtonRow/index.tsx (hunk 9 above)
[the remark this answers is not in the document]
Comment (aswallace) [reply packages/core/components/GlobalActionButtonRow/index.tsx:82]: Got it! I mainly meant the image/grid ones

File: packages/core/components/FileDetails/FileDetails.module.css (hunk 2 above)
Comment (aswallace) [reply packages/core/components/FileDetails/FileDetails.module.css:13]: Window staying the same size, trying to increase the size of the file details one (dragging the left bar of the file details panel towards the center of the screen)

File: packages/core/App.tsx [hunk 11]
```diff
@@ -27,11 +27,12 @@ export const ROOT_ELEMENT_ID = "root";
 initializeIcons();
 loadTheme({
     defaultFontStyle: {
-        fontFamily: "Roboto",
+        fontFamily: "Open sans, sans-serif",
```
Comment (kmitcham) [inline packages/core/App.tsx:30]: Roboto is an objectively cooler name

File: packages/web/src/components/Header/Header.module.css (hunk 7 above)
Comment (SeanLeRoy) [reply packages/web/src/components/Header/Header.module.css:50]: I'll note this down for part 2

File: packages/core/components/FileDetails/index.tsx (hunk 5 above)
Comment (SeanLeRoy) [reply packages/core/components/FileDetails/index.tsx:100]: This is part of the responsive UI design lyndsay mocked up, will include in part 2 or in a totally separate responsive design refactor

File: packages/core/components/FileDetails/FileDetails.module.css (hunk 4 above)
Comment (SeanLeRoy) [reply packages/core/components/FileDetails/FileDetails.module.css:184]: RIP, will fix

File: packages/core/components/GlobalActionButtonRow/index.tsx (hunk 9 above)
[the remark this answers is not in the document]
Comment (SeanLeRoy) [reply packages/core/components/GlobalActionButtonRow/index.tsx:82]: Ah forgot about that, yeah it was recommended since it might help show that we try to display photos ig

**Comment by BrianWhitneyAI** [comment]
main icon is all black. (Is this intentional?)
p1

**Comment by BrianWhitneyAI** [comment]
Divider section highlight range is very small and hard to select/drag

**Comment by BrianWhitneyAI** [comment]
The edit and delete icons for the filter are messy and dont have a hover 
Untitled

**Comment by BrianWhitneyAI** [comment]
The group by module icon is misaligned
Untitled_2

**Comment by BrianWhitneyAI** [comment]
If we gray out datasource for AICS FMS query then why do we need it.

**Comment by BrianWhitneyAI** [comment]
I think the text in the main explorer area (filenames…) is too close together and are not centered vertically to the background stripe 
Untitled_4

**Comment by BrianWhitneyAI** [comment]
Cannot delete last query.

**Comment by BrianWhitneyAI** [comment]
I think these should be uppercase 
Untitled_5

**Comment by BrianWhitneyAI** [comment]
Column separators are not vertically centered.
Untitled_6

**Comment by BrianWhitneyAI** [comment]
We used to have a bar divider after the recent annotations. Was there a reason for removing this?

**Review by BrianWhitneyAI — CHANGES_REQUESTED** [review]
lots of changes here! I think the alignment and spacing could use some love and the color scheme might be becoming too dark.

File: packages/core/components/QueryPart/QuerySort.tsx [hunk 12]
```diff
@@ -18,12 +18,22 @@ interface Props {
...
+                props.sort.order === SortOrder.ASC ? SortOrder.DESC : SortOrder.ASC;
+            const newSort = new FileSort(props.sort.annotationName, oppositeOrder);
+            dispatch(selection.actions.setSortColumn(newSort));
+        }
+    };
+
     return (
         <QueryPart
             title="Sort"
+            titleIconName={props.sort?.order === SortOrder.ASC ? "SortUp" : "SortDown"}
```
[the remark this answers is not in the document]
Comment (SeanLeRoy) [reply packages/core/components/QueryPart/QuerySort.tsx:33]: Fixed 👍

File: packages/core/components/FileDetails/FileDetails.module.css (hunk 2 above)
Comment (SeanLeRoy) [reply packages/core/components/FileDetails/FileDetails.module.css:13]: Left this for round 2 because a lot of that panel needs to be refactored

File: packages/core/components/QueryPart/QueryPartRow.module.css (hunk 3 above)
Comment (SeanLeRoy) [reply packages/core/components/QueryPart/QueryPartRow.module.css:55]: Fixed

File: packages/core/components/FileDetails/FileDetails.module.css (hunk 4 above)
Comment (SeanLeRoy) [reply packages/core/components/FileDetails/FileDetails.module.css:184]: Fixed

File: packages/core/components/TutorialTooltip/TutorialTooltip.module.css (hunk 6 above)
Comment (SeanLeRoy) [reply packages/core/components/TutorialTooltip/TutorialTooltip.module.css:38]: Fixed _I think_

File: packages/desktop/src/renderer/renderer.module.css
Comment (SeanLeRoy) [reply packages/desktop/src/renderer/renderer.module.css:1]: Removed

File: packages/core/styles/global.css (hunk 8 above)
Comment (SeanLeRoy) [reply packages/core/styles/global.css:23]: Having trouble replicating this - created an issue to check for styling differences on desktop since I there is at least 1-2 other things

File: packages/core/styles/global.css [hunk 13]
```diff
@@ -25,56 +24,41 @@ main {
 }
```
Comment (aswallace) [inline packages/core/styles/global.css:24]: Line above this: `--borders-light-grey` should change to `--medium-grey`

File: packages/web/src/components/Home/index.tsx [hunk 14]
```diff
@@ -16,33 +17,25 @@ export default function Home() {
...
-                    styles={{
-                        label: styles.buttonLabel,
-                        icon: styles.buttonIcon,
-                    }}
-                    ariaLabel="Upload file"
-                    iconProps={{ iconName: "Upload" }}
-                    title="Upload file"
-                    text="UPLOAD FILE"
-                />
+                <Link to="app">
```
Comment (aswallace) [inline packages/web/src/components/Home/index.tsx:20]: It seems like with the new button styles we might need to add `text-decoration: none;` to the link part
image

File: packages/core/components/TutorialTooltip/TutorialTooltip.module.css (hunk 6 above)
Comment (SeanLeRoy) [reply packages/core/components/TutorialTooltip/TutorialTooltip.module.css:38]: I'll note this for the issue targeted for adding responsive design 👍  good catch
